### PR TITLE
Issue #10 Add option for exteneded parameter docs

### PR DESCRIPTION
**Pull request by mikehaertl**
A naive implementation of #10.

Review/improvements very welcome

### CODE REVIEW

**Comment by mikehaertl** [comment]
@hrsh7th Could you have a look at this? I know you consider this plugin a bit of a hack but some still prefer it over lsp_signature.vim. So maybe we can tweak it just a little.

**Comment by hrsh7th** [comment]
Sorry! I missed this PR. I'll review it.

File: lua/cmp_nvim_lsp_signature_help/init.lua [hunk 1]
```diff
@@ -138,13 +153,22 @@ source._docs = function(self, signature, parameter_index)
...
+      else
+        label = doc
+      end
+    else
+      local s, e = string.find(label, self:_parameter_label(signature, signature.parameters[parameter_index]), 1, true)
+      if s and e then
+        local active = string.sub(label, s, e)
+        label = string.gsub(label, vim.pesc(active), '***' .. active .. '***')
+      end
     end
```
Comment (hrsh7th) [inline lua/cmp_nvim_lsp_signature_help/init.lua:172]: Thanks for your help!

I don't think you need the `extended_docs` option. Your implementation is correct. I just forgot to implement the parameter documentation.

Comment (mikehaertl) [reply lua/cmp_nvim_lsp_signature_help/init.lua:149]: Ok, I've removed the config part and instead replaced the old logic. I hope that's what you meant?

**Comment by mikehaertl** [comment]
Sure, why not. Usually the more information, the better.

**Comment by hrsh7th** [comment]
Thanks to your contribution, it has been further improved.

**Pull request closed without merging**